lib/msnodesqlv8/request.js

```diff
@@ -514,8 +514,11 @@ class Request extends BaseRequest {
           }
         })
 
-        req.on('rowcount', count => {
-          rowsAffected.push(count)
+        req.on('rowcount', rowCount => {
+          rowsAffected.push(rowCount)
+          if (this.stream && !this.paused) {
```

Comment:
This may be in place in other places, but I think the event should still be emitted if the stream is paused; if it's not emitted then it's effectively silently swallowed and if app developers depend on this, we need to emit it...

Reply:
Thanks, I was wondering about this as well. I've made the changes 👍